### PR TITLE
Add postgresql 9.4 to travis builds

### PR DESCRIPTION
**Pull request by billschaller**

### CODE REVIEW

**Comment by zeroedin-bill** [comment]
Goes in hand with #824

File: .travis.yml [hunk 1]
```diff
@@ -49,9 +52,16 @@ matrix:
       env: DB=pgsql POSTGRESQL_VERSION=9.2 # driver currently unsupported by HHVM
     - php: hhvm
       env: DB=pgsql POSTGRESQL_VERSION=9.3 # driver currently unsupported by HHVM
+    - php: hhvm
+      env: DB=pgsql POSTGRESQL_VERSION=9.4 # driver currently unsupported by HHVM
```
Comment (Ocramius) [inline .travis.yml]: Wondering if this still holds true considering https://github.com/PocketRent/hhvm-pgsql. Is HHVM really not supporting a single actually working RDBMS?

File: .travis.yml [hunk 2]
```diff
@@ -49,9 +52,16 @@ matrix:
...
       env: DB=pgsql POSTGRESQL_VERSION=9.1 # driver currently unsupported by HHVM
     - php: hhvm-nightly
       env: DB=pgsql POSTGRESQL_VERSION=9.2 # driver currently unsupported by HHVM
     - php: hhvm-nightly
       env: DB=pgsql POSTGRESQL_VERSION=9.3 # driver currently unsupported by HHVM
+    - php: hhvm-nightly
+      env: DB=pgsql POSTGRESQL_VERSION=9.4 # driver currently unsupported by HHVM
+
+addons:
+  postgresql: '9.4'
```
Comment (Ocramius) [inline .travis.yml]: Never saw this: what's that?

Comment (zeroedin-bill) [inline .travis.yml]: Its from the travis-ci devblog- 9.4 support is experimental, and this is
how you enable it apparently.

File: .travis.yml (hunk 1 above)
Comment (zeroedin-bill) [inline .travis.yml]: I wondered that myself. I would imagine this is no longer the case, but I
don't use hhvm.

File: .travis.yml (hunk 2 above)
Comment (zeroedin-bill) [inline .travis.yml]: @Ocramius see the PostgreSQL 9.4 section in [this post](http://docs.travis-ci.com/user/build-environment-updates/2015-02-03/)

**Pull request merged**